Makefile

```diff
@@ -1,22 +0,0 @@
-DOCKER=docker
```

Comment:
The makefile is welcome. But since we are calling the tests through the entry point directive, the makefile it's not particularly necessary for this particular feature.

Would be nice to have some make commands to run unit tests locally - even though the IDE does that.